ocaml/libs/http-lib/http_svr.ml

```diff
@@ -221,17 +221,23 @@ let response_redirect ?req s dest =
   in
   Unixext.really_write_string s (Http.Response.to_wire_string res)
 
-let response_file ?mime_content_type s file =
+let response_file ?mime_content_type ~add_hsts s file =
   let size = (Unix.LargeFile.stat file).Unix.LargeFile.st_size in
   let mime_header =
     Option.fold ~none:[]
       ~some:(fun ty -> [(Hdr.content_type, ty)])
       mime_content_type
   in
   let keep_alive = (Http.Hdr.connection, "keep-alive") in
+  let hsts =
+    if add_hsts then
+      [(Http.Hdr.hsts, "max-age=63072000")] (* max-age sets to 2yr *)
```

Comment:
Would be better to compute this as in `Printf.sprintf "max-age=%d" (60*60*24*365*2)` or similar to avoid magic constants.